tests/osg/Matrix.js

```diff
@@ -1,9 +1,19 @@
 define( [
     'qunit',
     'tests/mockup/mockup',
+    'osg/Light',
```

Comment:
it's not the place to add more high level tests. Test with the same class name should unittest the class. For higher level test with sample scene or other should be in another directory.